proto/rig/api/v1/cluster/service.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package api.v1.cluster;
+
+// The service to your cluster running Rig.
+service Service {
+  // Status returns information about the cluster
+  rpc Status(StatusRequest) returns (StatusResponse) {}
```

Comment:
Perhaps we should put this under "Config", where Status eventually could be "Is the registry available"